_posts/2024-12-23-cold-start-search.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title:  "Solving the cold start search problem in OpenSearch"
+authors:
+ - aswath
+ - allan
+ - kolchfa
+date: 2024-12-23
```

Comment:
Please update the blog publish date to: 
2025-01-07

Reply:
Done